docker/templates/Dockerfile.j2

```diff
@@ -124,7 +124,8 @@ RUN \
     mkdir /licenses/ && \
     mv /usr/share/logstash/NOTICE.TXT /licenses/NOTICE.TXT && \
     mv /usr/share/logstash/LICENSE.txt /licenses/LICENSE.txt && \
-    find /usr/share/logstash -type d -exec chmod g+s {} \; && \
+{% if image_flavor != 'ironbank' -%}  find /usr/share/logstash -type d -exec chmod g+s {} \; && \
+{% endif -%}
```

Comment:
I know it's merged already but for future reference, [looking at the docs](https://jinja.palletsprojects.com/en/3.1.x/templates/#if-expression) there seems to be a way to do inline if expression:

```j2
{{ 'Update' if files else 'Continue' }}
```